common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -29,6 +29,11 @@
 public final class CommonUtils {
   private static final Logger LOG = LoggerFactory.getLogger("");
 
+  /**
+   * Get the current time in milliseconds.
+   *
+   * @return current time in milliseconds
+   */
```

Comment:
@calvinjia Is there a convention for documenting `getter` methods, I feel that only the `@return` is needed.